apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -126,11 +126,13 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 		overflow: hidden;
 
 		&:hover,
-		&:focus {
+		&:focus,
```

Comment:
should we remove the pseudo `:focus` since it's going to be handled by the CSS class? I wouldn't like to see two selected thumbs despite it should impossible to get.

Reply:
We still need :focus to stay keyboard accessible (focus moves when you tab around). I would want it to be something else than the same blue border so I’ve raised it in the parent issue of this one.

Reply:
IC, thought we were handling `onFocus` as well